### ISSUE TITLE
Is it possible to add an operator that is both right-hand-side-only and can act as a normal one?

### ISSUE THREAD

**Issue by Yazwh0** [codingseb/ExpressionEvaluator#159]
So in 6502 asm we use '<' '>' and '^' to denote high, low and top bytes. 

eg
```
var a = #<$abcd ; equals 0xab
```

But I also need to be able to support greater\less than.

eg

```
var b = 1
var a = #<(b < 5 ? $abcd : $1234) ; should also equal 0xab
```

It feels like these are 'unary operators' but they seem to be handled in a very unique way, and not definable by extending the class?

Does anyone know if there is there a more cunning way, or do I need to hack up the Regex?

I know the original author had left this project -- which is a shame, as its really good! -- but hoping someone else might know?

**Comment by Yazwh0** [comment]
I think I have this working using unary operators in a new PR #160

**Comment by codingseb** [comment]
Your PR has been included in version 1.4.40.0.
Thanks for your work @Yazwh0
